main.js

```diff
@@ -0,0 +1,379 @@
+$(document).ready(function() { //adds click handlers after DOM loads
+    addClickHandler();
+    readyPageFunctions();
+});
+
+//Click handlers for each color
+function readyPageFunctions() {
+    $(".red").on("click", playerColor);
+    $(".blue").on("click", playerColor);
+    $(".gold").on("click", playerColor);
+    $(".green").on("click", playerColor);
+}
+
+function addClickHandler() {
+    $(".col0").on('click', function(){
+        columnNumber = 0;
+        addToGridArray();
+    });
+
+    $(".col1").on('click', function(){
+        columnNumber = 1;
+        addToGridArray();
+    });
+
+    $(".col2").on('click', function(){
+        columnNumber = 2;
+        addToGridArray();
+    });
+
+    $(".col3").on('click', function(){
+        columnNumber = 3;
+        addToGridArray();
+    });
+
+    $(".col4").on('click', function(){
+        columnNumber = 4;
+        addToGridArray();
+    });
+
+    $(".col5").on('click', function(){
+        columnNumber = 5;
+        addToGridArray();
+    });
+
+    $(".col6").on('click', function(){
+        columnNumber = 6;
+        addToGridArray();
+    });
+
+    $(".reset").on('click', function() {
+        resetGame();
+    });
+
+    $(".toggleAI").on('click', function() {
+        toggleAI();
+    });
+
+    $(".togglePlayerNumber").on('click', function() {
+        togglePlayerMode();
+    });
+}
+
+var playerSwitch = 2; //variable that switches between players one, two and three
+var playerOneColor = null; // store player one's image
+var playerTwoColor = null; // store player two's image
+var playerThreeColor = null; // store player three's image
+var victoryTrigger = 0; //var that becomes 1 when game victory, disables certain interactive functions;
+var gameArray = [[null, null, null, null, null, null, null], [null, null, null, null, null, null, null], [null, null, null, null, null, null, null], [null, null, null, null, null, null, null],
+    [null, null, null, null, null, null, null], [null, null, null, null, null, null, null], [null, null, null, null, null, null, null]];
+var columnNumber = null;
+var playerModeToggle = 1;
+
+function togglePlayerMode() {
+    $('.title').text("Third Player Summoned. Player One: Select Your Deity.")
+    $('.togglePlayerNumber').text("TWO PLAYER MODE");
+    playerModeToggle = 1 - playerModeToggle; //if playerModeToggle === 0; three player mode activates; else two player mode;
+    if (playerModeToggle === 1) {
+        $('.title').text("Third Player Sacrificed. Player One: Select Your Deity.");
+        $('.togglePlayerNumber').text("THREE PLAYER MODE");
+    }
+    toggleAICount = 0;
+}
+
+var toggleAISwitch = 1; //toggle that switches between player and AI
```

Comment:
Having global variables dispersed throughout your document is counter-intuitive. You should keep all these variables at the top of your document together, for easy reference.